lesson_six_hw/task_two_four.js

```diff
@@ -0,0 +1,23 @@
+
+// 4. Написать функцию, которая принимает на вход слово и проверяет, является ли это слово палиндромом
+
+function checkForPalindrome (word) {
+  const wordArr = word.split('');
+  const wordArrFlippedOver = word.split('').reverse();
+  let мessage;
+  let result = true;
+
+  for (i = 0; i < wordArr.length; i++) {
+    if (wordArr[i] != wordArrFlippedOver[i]) {
+      result = false;
+      break;
+    }
+  } 
+  if (result === true) {
+    message = 'This is a palindrome';
+  } else {
+    message = 'This is not a palindrome';
+  }
+  return message;
```

Comment:
```suggestion
  let мessage;
  let result = true;

  for (i = 0; i < wordArr.length; i++) {
    if (wordArr[i] != wordArrFlippedOver[i]) {
      result = false;
      break;
    }
  } 
return result
```
Better create messages outside the function, function should return only boolean value